src/instructlab/sdg/flows/synth_grounded_skills.yaml

```diff
@@ -0,0 +1,65 @@
+version: "1.0"
+block_configs:
+  - block_type: LLMBlock
+    block_config:
+      block_name: gen_contexts
+      config_path: configs/skills/contexts.yaml
+      output_cols:
+        - context
+    gen_kwargs:
+      temperature: 0.7
+      max_tokens: 2048
```

Comment:
```
                "gen_kwargs": {
                    "temperature": 0.7,
                    "max_tokens": 2048,
                    "n": self.ctx.num_instructions_to_generate,
                },
```

`n` lost here, but that's covered in another comment and I think it'll be the same fix

Reply:
resolved in this commit - https://github.com/instructlab/sdg/pull/86/commits/5a0b7a6f0e3d78dfec063b430b977601b8c54741